### DISCUSSION TITLE
No module named 'pysu2'

### DISCUSSION THREAD

**Discussion by jjywow** [su2code/SU2#1828] [Q&A] [answered]
Hello,

I installed Binary Executables 'SU2 v7.4.0 for windows' and setting the paths, which doesn't require compile with meson or ninja.

I'm doing tutorials. 'Static Fluid-Structure Interaction (FSI)' works just fine.

but when i doing tutorial 'Dynamic Fluid-Structure Interaction (FSI) using the Python wrapper and a Nastran structural model' ,

I encountered error as follows,
File "\SU2\bin\fsi_computation.py", line 41, in <module>
    import pysu2
ModuleNotFoundError: No module named 'pysu2'

How can I get module pysu2?

Thank you for your time reading this.

os: window

**Comment by Nicola-Fonzi** [comment]
Hi,

Thank you for posting this. When I created the tutorial, I compiled SU2 for my machine enabling the feature pywrapper. I do not know how the windows binaries are created and if they include the wrapping. @pcarruscag do you know something about this?

**Reply by pcarruscag** [reply]
The binaries do not have the python wrapper.

**Comment by Nicola-Fonzi** [comment, marked as answer]
Ok, thanks for the reply. @jjywow In this case you have to compile the code if you want to run the tutorial you mentioned, or any other that include the python wrapper.

My personal suggestion is to do it on a linux machine, with direct access to the hardware, so that you easily run parallel (i.e., not the WSL)

Please mark the conversation as resolved if this is clear, otherwise feel free to comment if you have further questions

**Reply by jjywow** [reply]
Hello, have you seen anyone did successively compile on window or WSL?
It seems I can't

**Reply by jjywow** [reply]
@Nicola-Fonzi Hello. Thanks to you I could carry out your tutorial 'Dynamic Fluid-Structure Interaction (FSI) using the Python wrapper and a Nastran structural model'. 
I like to conduct the simulation with different shape of wing to find flutter speed.
For different shape of wing, can I just changed the fluid mesh? or should I modify the f06 file as well?
If I have to modify the f06 file, what part should I change? I'm not familiar with Nastran file
Thanks

**Reply by Nicola-Fonzi** [reply]
The Nastran mesh and the fluid mesh are not matching. If you look closely at the Nastran mesh you see that it is a simple rectangle. The code will take care of the interpolation creating the connection between the nonmatching meshes. If you change the airfoil (and the aerodynamic mesh correspondingly) you do not need to change the structural mesh. The latter statements only holds true as long as the chord of the new airfoil is similar to the previous one

**Reply by jjywow** [reply]
Thank you very much!
